### PR TITLE
Improved textEdit placeholder

### PR DESCRIPTION
**Pull request by zneix**
Pull request checklist:

- ~~[ ] `CHANGELOG.md` was updated, if applicable~~
not applicable

# Description

I applied some suggestions about placeholder text being a bit too 'easily readable' and made it slightly lighter to not mix with color of system messages and message timestamps.
I also read that tooltip might've been causing some issues on non-twitch channels, so I removed it (though, it only makes sense in twitch channels anyway).

### CODE REVIEW

File: src/widgets/splits/Split.cpp [hunk 1]
```diff
@@ -298,6 +303,19 @@ void Split::onAccountSelected()
     }
 
     this->input_->ui_.textEdit->setPlaceholderText(placeholderText);
+
+    this->updateTooltipColor();
+    this->signalHolder_.managedConnect(this->theme->updated, [this]() {
+        this->updateTooltipColor();  //
```
Comment (23rd) [inline src/widgets/splits/Split.cpp:309]:
```suggestion
        this->updateTooltipColor();
```

Comment (pajlada) [reply src/widgets/splits/Split.cpp:309]: Single line lambxas will be disabled soon tm so the code is good as is

**Review by pajlada — CHANGES_REQUESTED** [review]
Could you update the existing changelog entry to include this pr number?

**Comment by zneix** [comment]
> Could you update the existing changelog entry to include this pr number?

Sure thing, resolved in e99b353d

**Review by pajlada — APPROVED** [review]

**Pull request merged**